apps/ledger-live-desktop/package.json

```diff
@@ -16,7 +16,7 @@
   "license": "MIT",
   "private": true,
   "main": "./.webpack/main.bundle.js",
-  "version": "3.0.0-nightly.13",
+  "version": "2.52-nightly.13",
```

Comment:
Last version before the major bump was `2.52-nightly.11`, could you restore this instead of 13?